.github/workflows/contracts-deploy.yml

```diff
@@ -85,27 +87,35 @@ jobs:
         run: node .github/helpers/contracts/updateActiveContractsJSON.js
       - name: Update Subgraph manifests
         run: node .github/helpers/subgraph/updateManifest.js
+      - name: Commit changes
+        run: |
+          git fetch
+          git pull
+          git config --global user.name "Arabot-1"
+          git config --global user.email "arabot-1@users.noreply.github.com"
+          git add active_contracts.json
+          git add packages/contracts/Releases.md
+          git add ./packages/subgraph/	
+          git commit -am "Updates files with deployed contract addresses"
+          git push
       - name: create Dummy DAOs erc20
-        run: node .github/helpers/contracts/dummy-dao/createDao.ts rinkeby ${{ secrets.ETH_KEY }} erc20
+        run: ts-node .github/helpers/contracts/dummy-dao/createDao.ts $NETWORK ${{ secrets.ETH_KEY }} erc20
```

Comment:
from where does it get the $NETWORK from? 
if you like to do it per network you need to move these scripts into the `build-deploy` step or setup a matrix job here.
I recommend the matrix job. See `build-deploy` as a reference